api/nakadi-event-bus-api.yaml

```diff
@@ -1648,6 +1698,25 @@ definitions:
         description: |
           Offset of the event being pointed to.
 
+  ShiftedCursor:
```

Comment:
`"OffsetCursor"` ? That would make the field name `"offset"`. Just a thought, to reduce the number of terms in the API's design.